lms/envs/common.py

```diff
@@ -3294,6 +3294,11 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     'DEFAULT_RENDERER_CLASSES': (
         'rest_framework.renderers.JSONRenderer',
     ),
+    'DEFAULT_AUTHENTICATION_CLASSES': [
+        'edx_rest_framework_extensions.auth.jwt.authentication.JwtAuthentication',
+        'rest_framework.authentication.SessionAuthentication',
+        'rest_framework.authentication.BasicAuthentication'
```

Comment:
I don't know that we really want `BasicAuthentication` as a default. Some options:
1. Now or in a follow-up, add monitoring to `BasicAuthentication` to see if we can drop it as a default. And/or,
2. Add a comment here noting that we are adding `BasicAuthentication` for backward compatibility with DRFs defaults, but we don't necessarily want it as a default.

Reply:
What are your thoughts on a separate PR for observability around this that could help with other future auth deployments?

One option would be to add a custom attribute near https://github.com/openedx/edx-drf-extensions/blob/aa649e9ab13273a0624bfb623d4c80112631b601/edx_rest_framework_extensions/middleware.py#L148 in this middleware to `successful_authenticator` for DRF.
```
(Pdb) request.successful_authenticator.__class__.__module__
'edx_rest_framework_extensions.auth.session.authentication'
(Pdb) request.successful_authenticator.__class__.__name__
'MySessionAuthentication'
```

Separately, do you think edx.org uses `BasicAuthentication` anywhere where this could break if we leave it off? Observability would just be to confirm that we are not.